vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/TextValidationSteps.java

```diff
@@ -151,7 +151,7 @@ private List<WebElement> findElements(SearchContext searchContext, By locator)
      * @param text Text value
      * @return <code>true</code> if text does not exist, otherwise <code>false</code>
      */
-    @Then("the text '$text' does not exist")
+    @Then("text `$text` does not exist")
     public boolean textDoesNotExist(String text)
```

Comment:
the code of this step uses deprecated API underneath, it makes sense:
- to keep the code of deprecated step as is (only mark in properly for deprecation)
- add new step with updated syntax, refactor it to do not use deprecated API-s

Reply:
could you please specify deprecated code?